### ISSUE TITLE
Are you planning to support tensorflow 2?

### ISSUE THREAD

**Issue by Crispy13** [titu1994/Keras-Group-Normalization#9]
It would be glad if GN is compatible with tensorflow 2

**Comment by titu1994** [comment]
Tensorflow add-ons already has group normalization.
